lisa/sut_orchestrator/azure/common.py

```diff
@@ -561,6 +561,11 @@ def from_node_runbook(cls, runbook: AzureNodeSchema) -> "AzureNodeArmParameter":
         if "vhd" in parameters:
             parameters["vhd_raw"] = parameters["vhd"]
             del parameters["vhd"]
+        if "purchase_plan" in parameters:
```

Comment:
It should follow other fields, to be stripped in `AzureNodeSchema.__post_init__`.

Reply:
@squirrelsc I have modified the PR to support vhds with purchase plans. Please review.